spec/system/user_creates_an_article_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "User creates an article" do
+  it "shows their amazing content" do
+    visit root_path
+    fill_in "Title", with: "This is my awesome title"
+    fill_in "Content", with: "My awesome content"
+    click_button "Submit"
+
+    expect(page).to have_content "Article was successfully created"
+    article = Article.last
+    expect(page).to have_content "Article ##{article.id}"
+    expect(page).to have_content article.title
+    expect(page).to have_content article.content
+  end
+
+  context "user provides wrong form data" do
+    it "shows an error on the form" do
+      visit root_path
+
+      click_button "Submit"
+
+      expect(page).to have_content "New article"
+      expect(page).to have_content "2 errors prohibited this article from being saved"
+      expect(page).to have_content "Title can't be blank"
+      expect(page).to have_content "Content can't be blank"
+    end
+  end
+
+  xit "shows a success message" do
+    visit root_path
+    fill_in "Title", with: "This is my awesome title"
+    fill_in "Content", with: "My awesome content"
+    click_button "Submit"
+
+    expect(page).to redirect_to(articles_path)
+    expect(flash[:notice]).to eq('Article was successfully created.')
+  end
```

Comment:
Do we still need this test? We're not running it at the moment. If we need it, we should make sure it passes. If we don't need it, we should remove it from the file.